src/main/java/org/csanchez/jenkins/plugins/kubernetes/AllContainersRunningPodWatcher.java

```diff
@@ -68,7 +83,13 @@ boolean areAllContainersRunning(Pod pod) {
         }
         for (ContainerStatus containerStatus : containerStatuses) {
             if (containerStatus != null) {
-                if (containerStatus.getState().getWaiting() != null) {
+                ContainerStateWaiting waitingState = containerStatus.getState().getWaiting();
+                if (waitingState != null) {
+//                    String waitingStateMsg = waitingState.getMessage();
```

Comment:
If you are not changing the behavior of this method, better to revert all changes to it.